Makefile

```diff
@@ -18,7 +18,7 @@
 # Configuration Variables
 # ========================
 
-PYTEST_ARGS ?= -v  # Override with e.g. PYTEST_ARGS="-vv --tb=short"
+PYTEST_ARGS ?= -v -x  # Override with e.g. PYTEST_ARGS="-vv --tb=short"
```

Comment:
-x to exit test when ctrl-c

```
  -x, --exitfirst       exit instantly on first error or failed test.
```
https://docs.pytest.org/en/6.2.x/reference.html#command-line-flags